server/server.js

```diff
@@ -81,6 +84,39 @@ app.post('/blogs', async (req, res) => {
 
 
 
+//MEMO TO SELF: TWO METHODS SO CHECK WHICH ONE WORKS (OR WORKS BETTER) AS NOT SURE IF I HAVE TO DO A GET (WITH MULTIPLE PARAMS) OR JUST DO A POST REQUEST SO I CAN USE A BODY//
+/*POST method - for acquiring song infos*/
+app.post('/song', async (req, res) => {
+    try { 
+        const { artistName } = req.body;
+        const { trackName } = req.body;
+
+        const response = await fetch(`http://ws.audioscrobbler.com/2.0/?method=track.getInfo&api_key=LastFM_Key&artist=${artistName}&track=${trackName}}&format=json`)
+        const body = await response.json();
+
+        return res.send(body);
+    } catch (error) {
+        console.error(error.message)
+    }
+})
+/*GET method*/
+app.get('/song/:artistName/:trackName', (req, res) => {
+    try {
+        const { artistName } = req.params.artistName;
+        const { trackName } = req.params.trackName;
```

Comment:
```suggestion
        const { artistName, trackName } = req.params;
```
Similarly, you can do that here too! But if you don't want to, I think the code needs to exclude the `.artistName` and `.trackName` since you are already retrieving it 
```suggestion
    const { artistName } = req.params;
    const { trackName } = req.params;
```